[{"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/MI_FEMA_Luce_Schoolcraft_2019/ept.json"], "error": ""}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/NC_FranklinCo_2014/ept.json"], "error": "PDAL: readers.ept: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/NC_FranklinCo_2014/ept.json\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/NC_Phase5_Haywood_2017/ept.json"], "error": "PDAL: readers.ept: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/NC_Phase5_Haywood_2017/ept.json\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/NC_Phase5_Henderson_2017/ept.json"], "error": ""}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/NE_Niobrara_Topobathy_2018/ept.json"], "error": ""}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/NE_Northeast_1_2020/ept.json"], "error": ""}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/NJ_South_Jersey_FEMA_2018/ept.json"], "error": "PDAL: readers.ept: Error reading tile: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/NJ_South_Jersey_FEMA_2018/ept-data/1-1-0-0.laz\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/NM_Northeast_B4_2018/ept.json"], "error": "PDAL: readers.ept: Error reading tile: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/NM_Northeast_B4_2018/ept-data/2-0-2-2.laz\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/NM_SouthEast_B5_2018/ept.json"], "error": "PDAL: readers.ept: Error reading tile: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/NM_SouthEast_B5_2018/ept-data/2-2-1-1.laz\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/OR_Malheur_USFS_2010/ept.json"], "error": "PDAL: readers.ept: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/OR_Malheur_USFS_2010/ept.json\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/PA_WesternPA_4_2019/ept.json"], "error": ""}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/SC_OrangeburgCo_2008/ept.json"], "error": ""}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/SC_SavannahPeeDee_4_2019/ept.json"], "error": "PDAL: readers.ept: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/SC_SavannahPeeDee_4_2019/ept.json\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/SD_EasternSD_1_2020/ept.json"], "error": ""}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/SD_EasternSD_2_2020/ept.json"], "error": "PDAL: readers.ept: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/SD_EasternSD_2_2020/ept.json\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/SD_NRCS_QSI_A1_2017/ept.json"], "error": "PDAL: readers.ept: Error reading tile: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/SD_NRCS_QSI_A1_2017/ept-data/1-1-0-0.laz\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/TX_Pecos_Dallas_B2b_2018/ept.json"], "error": "PDAL: readers.ept: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/TX_Pecos_Dallas_B2b_2018/ept.json\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_AK_POW_P2_2018_LAS_2019/ept.json"], "error": "PDAL: readers.ept: Error reading tile: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_AK_POW_P2_2018_LAS_2019/ept-data/3-3-6-4.laz\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_AK_POW_P2_TL_2018_LAS_2019/ept.json"], "error": ""}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_AL_MobileCo_2014_LAS_2016/ept.json"], "error": "PDAL: readers.ept: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_AL_MobileCo_2014_LAS_2016/ept.json\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_CA_FEMA_R9_Mendocino_HF_2017_LAS_2018/ept.json"], "error": "PDAL: readers.ept: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_CA_FEMA_R9_Mendocino_HF_2017_LAS_2018/ept.json\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_CA_NoCAL_Wildfires_B2_2018/ept.json"], "error": "PDAL: readers.ept: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_CA_NoCAL_Wildfires_B2_2018/ept.json\n\n"}, {"error": "Could not create geometry because of errors while reading input."}, {"error": "Could not create geometry because of errors while reading input."}, {"tile": "USGS_LPC_WA_Western_North_2016_LAS_2018", "error": "Expecting value: line 1 column 1 (char 0)"}, {"tile": "USGS_LPC_WA_Western_South_2016_LAS_2018", "error": "Expecting value: line 1 column 1 (char 0)"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/WI_Oshkosh_3Rivers_FondDuLac_TL_2018/ept.json"], "error": "PDAL: readers.ept: Error reading tile: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/WI_Oshkosh_3Rivers_FondDuLac_TL_2018/ept-data/0-0-0-0.laz\n\n"}]